doc/SeparatingVttabletMysql.md

```diff
@@ -25,7 +25,7 @@ The following adjustments need to be made to VTTablet command line parameters:
   `-db\_socket` parameters.
 
 * Disable restores / backups, by not passing any backup command line
-  parameters. Specifically, `-restore_from_backup` and
+  parameters. Specifically, `--restore_from_backup` and
```

Comment:
I didn't want to explode this PR (also it's extremely tangential to this unit of work) by updating everything here, so I filed https://github.com/vitessio/vitess/issues/11058